docs/_config.yml

```diff
@@ -0,0 +1,48 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: Demes manual
+author: PopSim Consortium
+copyright: "2021"
+only_build_toc_files: true
+# logo: logo.png
+
+execute:
+  execute_notebooks: cache
+
+repository:
+  url: https://github.com/popsim-consortium/demes-python
+  branch: main
+  path_to_book: docs
+
+html:
+  use_issues_button: true
+  use_repository_button: true
+  use_edit_page_button: true
+  # Do not edit this - the version placeholder is replaced by the
+  # current version during a distribution build in the Makefile
+  extra_navbar: demes __DEMES_VERSION__
+  extra_footer: demes __DEMES_VERSION__
+
+sphinx:
+    extra_extensions:
+    - sphinx.ext.autodoc
+    - sphinx.ext.todo
+    - sphinx.ext.viewcode
+    - sphinx.ext.intersphinx
+    - sphinx_issues
+
+    config:
+      issues_github_path: popsim-consortium/demes-python
+      todo_include_todos: true
+      intersphinx_mapping:
+        python: ["https://docs.python.org/3/", null]
+        spec: ["https://popsim-consortium.github.io/demes-spec-docs/main/", null]
+        stdpopsim: ["https://stdpopsim.readthedocs.io/en/stable", null]
```

Comment:
Do we need stdpopsim and msprime intersphinx mappings? We shouldn't really be depending on their APIs, right?

Reply:
Just for the `demes.convert` functions. How about I just remove that docs page now though?

Reply:
Yes, I think so. There's no point in documenting something we expect to remove immediately and don't want people to use.